src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx

```diff
@@ -44,9 +47,11 @@ export const ScheduleItem: React.FunctionComponent<PropsType> = ({
             <div>
               <b>{event.name}</b>
             </div>
-            <div>
-              by <b>{event.host}</b>
-            </div>
+            {event.host && (
+              <div>
+                by <b>{event.host}</b>
+              </div>
+            )}
```

Comment:
Why is this change needed here? Is the `host` not always set?

Reply:
We don't want to use `<b>` tags in general. Either use `<strong>`, or even better, following a similar pattern as used in `src/components/templates/PartyMap/components/RoomModalOngoingEvent/RoomModalOngoingEvent.tsx` with the `<span>` and a semanticly BEM named class.

Reply:
`host` conditional is removed, and `<b>` tags removed as in`RoomModalOngoingEvent.tsx`